modules/ref-dashboard-configuration-options.adoc

```diff
@@ -0,0 +1,41 @@
+
+.Dashboard feature configuration options
+[cols="20%,5%,75%","header"]
+|===
+| Feature        | Default |     Description
+| `dashboardConfig:enablement` | `true` | Enables admin users to add applications to the {productname-short} dashboard *Application → Enabled* page. To disable this ability, set the value to `false`.
+| `dashboardConfig:disableInfo` | `false` | On the *Applications→ Explore* page, when a user clicks on an application tile, an information panel opens with more details about the application. To disable the information panel for all applications on the *Applications→ Explore* page , set the value to `true`.          
+| `dashboardConfig:disableSupport` | `false` | Shows the *Support* menu option when a user clicks the Help icon in the dashboard toolbar. To hide this menu option, set the value to *true*.
+| `dashboardConfig:disableClusterManager` | `false` | Shows the *Settings → Cluster settings* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
+| `dashboardConfig:disableTracking` | `true`	| Allows Red Hat to collect data about {productname-short} usage in your cluster. To enable data collection, set the value to `false`. You can also set this option in the {productname-short} dashboard interface from the *Settings → Cluster settings* navigation menu.	
+| `dashboardConfig:disableBYONImageStream` | `false`  | Shows the *Settings → Notebook image settings* option in the dashboard navigation menu.  To hide this menu option, set the value to `false`.
+| `dashboardConfig:disableISVBadges` | `false`  | Shows the label on a tile that indicates whether the application is  “Red Hat managed”, “Partner managed”, or “Self-managed”. To hide these labels, set the value to `true`. 
+| `dashboardConfig:disableUserManagement` | `false` | Shows the *Settings → User management* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.                                                
+| `dashboardConfig:disableProjects` | `false` | Shows the *Data Science Projects* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.                                              
+ | `dashboardConfig:disablePipelines` | `false` | Shows the *Data Science Pipelines* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.                                             
+| `dashboardConfig:disableModelServing` | `false` | Shows the *Model Serving* option in the dashboard navigation menu and in the list of components for the data science projects. To hide *Model Serving* from the dashboard navigation menu and from the list of components for data science projects, set the value to `true`.       
+| `dashboardConfig:disableProjectSharing` | `false` | Allows users to share access to their data science projects with other users.  To prevent users from sharing data science projects, set the value to true.  
```

Comment:
- Looks like a double space before 'To prevent'.
- Can put 'true' in backticks.